maven/lib/dependabot/maven/file_parser.rb

```diff
@@ -44,8 +46,47 @@ def parse
         dependency_set.dependencies
       end
 
+      sig { returns(Ecosystem) }
+      def ecosystem
+        @ecosystem ||= T.let(
+          Ecosystem.new(
+            name: ECOSYSTEM,
+            package_manager: package_manager,
+            language: language
+          ),
+          T.nilable(Ecosystem)
+        )
+      end
+
       private
 
+      sig { returns(Ecosystem::VersionManager) }
+      def package_manager
+        @package_manager ||= T.let(
+          PackageManager.new(maven_version),
+          T.nilable(Dependabot::Maven::PackageManager)
+        )
+      end
+
+      sig { returns(T.nilable(Ecosystem::VersionManager)) }
+      def language
```

Comment:
@kbukum1 I'm not completely sure what the language should be in the case of maven. Is it ruby or java. As far as I'm aware maven is used to manage java dependencies but we never touch the java part or set the java version etc.

Reply:
Maven is a package manager for Java so the language is Java

Reply:
Language should be Java.